### ISSUE TITLE
Add layer for dam points location to Layer TOC

### ISSUE THREAD

**Issue by jwhaney** [TNRIS/lake-gallery#45]
Points should be available in the TOC. Will be simple point symbology, orange in color, and located at the center of the dam.

**Comment by jwhaney** [comment]
Need some clarification; conflict with mockup.

**Comment by jwhaney** [comment]
@LaurenKirk @csidenblad, so anything in the requirements doc related to the Layer TOC can I close or disregard?

**Comment by LaurenKirk** [comment]
@jwhaney yes,  sorry, should have updated this in req doc.  if we can add a dam point button to the new time slider tool that would be cool.  Our aim is to move away/remove the TOC

**Comment by jwhaney** [comment]
ok then this issue is still relevant. i'll keep it in tier 3 for the future once the layer is created.

__note__: the new layer will reside in the new transparent time slider / tool dialogue shown in the mockup

**Comment by jwhaney** [comment]
documenting this for future.
will eventually add a checkbox to the time slider tool with an option to turn off or on the dam points once this data is finalized/received. this will be similar to the lake outline checkbox in the pic below.

![image](https://user-images.githubusercontent.com/13525011/58904691-a4a1de00-86cd-11e9-91bb-5ae654609067.png)
